app/src/main/java/com/github/se/polyfit/ui/screen/GraphScreen.kt

```diff
@@ -0,0 +1,126 @@
+      }) { padding ->
+        Column(
+            horizontalAlignment = Alignment.CenterHorizontally,
+            modifier = Modifier.fillMaxSize().testTag("GraphScreenColumn").padding(padding)) {
+              Box(modifier = Modifier.fillMaxWidth().weight(1f)) {
+                Column(
+                    modifier = Modifier.fillMaxSize(),
+                    horizontalAlignment = Alignment.CenterHorizontally,
+                    verticalArrangement = Arrangement.SpaceAround) {
+                      if (!isTestEnvironment) {
+                        LineChart(
+                            modifier = Modifier.fillMaxSize().testTag("LineChart"),
+                            lineChartData = lineChartData(DataToPoints(), DateList()))
+                      } else {
+                        Spacer(modifier = Modifier.fillMaxSize().testTag("LineChartSpacer"))
+                      }
+                    }
+              }
+              Spacer(modifier = Modifier.height(5.dp))
+              Row(
+                  modifier = Modifier.fillMaxWidth().padding(5.dp),
+                  horizontalArrangement = Arrangement.SpaceBetween) {
+                    searchText?.let {
+                      OutlinedTextField(
+                          value = it,
+                          onValueChange = viewModel::onSearchTextChanges,
+                          modifier = Modifier.fillMaxWidth(0.35f).testTag("GraphScreenSearchBar"),
+                          placeholder = { Text(text = "Search") },
+                          shape = MaterialTheme.shapes.large)
+                    }
+                    DropDownMenu(
+                        isExpanded,
+                        SortPoints.toList(),
+                        sortedBy,
+                        onItemSelect = { sortAttributeName ->
+                          val sortAttribute = SortPoints.valueOf(sortAttributeName)
+                          viewModel.updateSort(sortAttribute)
+                        })
+                  }
+              Spacer(modifier = Modifier.height(5.dp))
+              HorizontalDivider(thickness = 1.dp, color = MaterialTheme.colorScheme.inversePrimary)
+              LazyColumn(modifier = Modifier.fillMaxWidth().weight(1f).testTag("ElementsList")) {
+                graphData?.let {
+                  items(it) { data ->
+                    Spacer(modifier = Modifier.height(5.dp))
+                    Row(
+                        modifier = Modifier.fillMaxWidth().testTag("ElementsRow"),
+                        horizontalArrangement = Arrangement.SpaceBetween) {
+                          Column(
+                              horizontalAlignment = Alignment.Start,
+                              modifier = Modifier.padding(start = 4.dp)) {
+                                Text(
+                                    text = "${data.kCal} kCal", modifier = Modifier.testTag("kcal"))
```

Comment:
If we want to use values in `strings.xml`, how come we don't use it here? I believe you can pass arguments?

Reply:
it seemed simpler as the data isn't fixed

Reply:
Even though the data is not fixed it still follows a format, and then you can use the strings.xml and pass a variable in. I think its better to either use this for everything or nothing at all, since it doesn't make sense to use it conditionally